### PR TITLE
Fixed normalisation issue in EXT/NON

### PR DESCRIPTION
**Pull request by MikeEntwistle**
This doesn't affect any results, but the normalisation
being printed to the screen in the real time propagation
is wrong, and therefore misleading. The fix checks that
the wavefunction is correctly normalised at each time
step.

### CODE REVIEW

File: iDEA/EXT1.py [hunk 1]
```diff
@@ -218,7 +218,8 @@ def main(parameters):
             # Normalise the wavefunction 
             norm = npla.norm(wavefunction)*np.sqrt(pm.space.delta)
             wavefunction /= norm
-            string = 'EXT: t = {:.5f}, normalisation = {}'.format(i*pm.sys.deltat, norm**2)
+            norm = npla.norm(wavefunction)*np.sqrt(pm.space.delta)
```
Comment (ltalirz) [inline iDEA/EXT1.py]: no need to compute the norm again (see line 219)

File: iDEA/NON.py [hunk 2]
```diff
@@ -223,7 +223,8 @@ def main(parameters):
                 # Normalise the wavefunction 
                 norm = npla.norm(wavefunction)*np.sqrt(pm.space.delta)
                 wavefunction /= norm
-                string = 'NON: t = {:.5f}, normalisation = {}'.format(i*pm.sys.deltat, norm**2)
+                norm = npla.norm(wavefunction)*np.sqrt(pm.space.delta)
+                string = 'NON: t = {:.5f}, normalisation = {}'.format(i*pm.sys.deltat, norm)
```
Comment (ltalirz) [inline iDEA/NON.py]: same here

**Comment by MikeEntwistle** [comment]
The wave function changes after the first renormalisation, so the second
renormalisation is to check that the wave function is correctly normalised.

**Comment by ltalirz** [comment]
Hm... Perhaps I misunderstood the purpose of the printout but with your changes aren't you checking a trivial mathematical identity at every timestep?
If you want to make sure that the line for computing the norm does what it is supposed to, why not wrap it in a function and write a unit test for that function?

**Comment by ltalirz** [comment]
Fine with this - still, I think it's unnecessary to compute `npla.norm(wavefunction)` twice here

**Pull request merged**